src/components/CalendarDay.jsx

```diff
@@ -64,7 +67,7 @@ export default class CalendarDay extends React.Component {
         onMouseLeave={e => this.onDayMouseLeave(day, e)}
         onClick={e => this.onDayClick(day, e)}
       >
-        {day.format('D')}
+        {renderDay ? renderDay(day) : day.format('D')}
```

Comment:
it would be better to have the default renderDay function as `(day) => day.format('D')` and just call it always, no need for the if statement